flexinput/src/main/java/com/lytefast/flexinput/fragment/PhotosFragment.kt

```diff
@@ -49,10 +50,16 @@ open class PhotosFragment : PermissionsFragment() {
     return view?.apply {
       recyclerView = findViewById(R.id.list)
 
-      val photoAdapter = PhotoCursorAdapter(context.contentResolver, selectionCoordinator!!)
+      val display = activity?.windowManager?.defaultDisplay
+      val size = Point()
```

Comment:
nit: would like an apply/run here, but up to you

Reply:
and I don't mind it as it's written but would like it to be called `displaySize` or `windowSize`, `size` is vaguebook af